cabal.haskell-ci

```diff
@@ -0,0 +1,10 @@
+ghc-head: True
+cabal-check: False
+no-tests-no-benchmarks: False
+unconstrained: False
+local-ghc-options: -Werror
+
+constraint-set embed-files
+  ghc: ==8.6.*
```

Comment:
I'm impressed by the fact that `haskell-ci` can limit this constraint set to just GHC 8.6. That being said, the fact that I did so in the original version was a crude hack. I'd actually want to run this on //all// versions of GHC that are tested here. (In other words, you can delete this line.)

Reply:
FWIW, for now the `ghc: ` field as required, but it can be changed to optional with "all ghc" as the default.